google-spreadsheet-config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file shouldn't be at root level but inside `ext` folder... See other existent `config.json` files for reference...